src/hotspot/share/opto/partialEscape.hpp

```diff
@@ -176,4 +189,15 @@ class PEAState {
 #endif
 };
 
+class MergeProcessor {
```

Comment:
Should we call this `PEAMergeProcessor`? MergeProcessor feels a too generic

Reply:
reasonable.